sdk/examples/custom_assertion.rs

```diff
@@ -1,89 +0,0 @@
-// Copyright 2022 Adobe. All rights reserved.
```

Comment:
Not a blocker for this PR, but let's make sure that there's an example for somebody building something like the [identity assertion](https://github.com/scouten-adobe/cawg-identity-core/) to build upon. (Maybe that will be a canonical example?)

Reply:
Yes, I'm working on updated examples for CBOR and JSON assertions, and it would be a good idea to add an identity example!